src/components/SignUpForm/index.tsx

```diff
@@ -0,0 +1,45 @@
+import ButtonLarge from '../shared/ButtonsLarge';
+import Logo from '../shared/Icons/logo';
+import InputField from '../shared/InputFields';
+import Link from '../shared/Links/links';
+
+const SignUpForm = () => {
+  return (
+    <div className='flex-col box-content items-center h-[34.75rem] w-[22.5rem] bg-indigo border-4m flex bg-white rounded-lg mt-16 ml-[7.438rem]'>
+      <Logo className='h-[1.875rem] w-[10.813rem] mt-12 mb-9' />
+      <InputField text='Email' placeholder='Type your email' variant='simple' required={true} />
+      <InputField
+        text='Full Name'
+        placeholder='Type your full name'
+        variant='simple'
+        required={false}
+      />
+      <InputField
+        text='Password'
+        placeholder='Type your password'
+        variant='password'
+        required={false}
```

Comment:
Does the input by default have the `required` as true? It seems kinda repetitive having to write everytime `required={false}`